ansible.cfg

```diff
@@ -1,6 +1,7 @@
 [defaults]
 roles_path = vendor/roles
 force_handlers = True
+vault_password_file = .vault_pass.example
```

Comment:
I had naively hoped that Ansible would just skip using `vault_password_file` if nothing was encrypted so that this could be set to `.vault_pass`.